site/en/community/mailing-lists.md

```diff
@@ -2,7 +2,7 @@
 
 As a community, we do much of our collaboration on public mailing lists. Please
 note that if you're looking for help using TensorFlow,
-[Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow) and
+[Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow), [TensorFlow Forum](https://discuss.tensorflow.org/) and
```

Comment:
Thanks, how about we reorder SO and TF Forum:

```suggestion
[TensorFlow Forum](https://discuss.tensorflow.org/),
[Stack Overflow](https://stackoverflow.com/questions/tagged/tensorflow), and
```

Reply:
That's also fine